### PR TITLE
Make log broadcaster backfill depth configurable

### PR DESCRIPTION
**Pull request by RyanRHall**
Addressed [#172492303](https://www.pivotaltracker.com/story/show/172492303)

### CODE REVIEW

File: core/store/orm/schema.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ type ConfigSchema struct {
 	GasUpdaterEnabled               bool            `env:"GAS_UPDATER_ENABLED" default:"false"`
 	JSONConsole                     bool            `env:"JSON_CONSOLE" default:"false"`
 	LinkContractAddress             string          `env:"LINK_CONTRACT_ADDRESS" default:"0x514910771AF9Ca656af840dff83E8264EcF986CA"`
+	LogBackfillDepth                string          `env:"LOG_BACKFILL_DEPTH" default:"10"`
```
Comment (se3000) [inline core/store/orm/schema.go:45]: Maybe rename it BLOCK_BACKFILL_DEPTH, or give it something to indicate what the unit is that the setting is measured in?

Comment (RyanRHall) [reply core/store/orm/schema.go:45]: :+1: yeah that sounds better to me

**Comment by tyrion70** [comment]
bors merge

**Comment by tyrion70** [comment]
bors merge

**Comment by kaleofduty** [comment]
bors try

**Comment by tyrion70** [comment]
bors merge

**Comment by RyanRHall** [comment]
bors merge

**Pull request merged**